frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx

```diff
@@ -89,13 +92,19 @@ const WalletDetails: React.FC = () => {
     });
 
     useEffect(() => {
+        void dispatch(loadTransactions());
```

Comment:
same

Reply:
fix in [https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/367/commits/41d8ea7e508c7c45c1cfa09406fa3012e4fce354](url)